trio/_core/_result.py

```diff
@@ -30,19 +30,4 @@ def capture(cls, sync_fn, *args):
 # ensure that isinstance(Value(), Result)/issubclass(Value, Result) and etc
 # don't break
 Result.register(Value)
-Result.register(Error)
-
-__deprecated_attributes__ = {
-    "Result":
-        _deprecate.DeprecatedAttribute(
-            Result, "0.5.0", issue=494, instead="outcome.Outcome"
-        ),
-    "Value":
-        _deprecate.DeprecatedAttribute(
-            Value, "0.5.0", issue=494, instead="outcome.Value"
-        ),
-    "Error":
-        _deprecate.DeprecatedAttribute(
-            Error, "0.5.0", issue=494, instead="outcome.Error"
-        )
-}
+Result.register(Error)
```

Comment:
Ugh. Could you please keep the newline?

Reply:
Whoops, missed that. Surprised yapf didn't catch it.